src/main/java/com/thealgorithms/misc/TrappingRainWater.java

```diff
@@ -16,7 +16,7 @@
  */
 public class TrappingRainwater {
```

Comment:
Your class is called `TrappingRainwater`, you should rename yout file to `TrappingRainwater.jave` instead :)